src/main/java/it/pagopa/ecommerce/commons/client/NpgClient.java

```diff
@@ -394,13 +394,15 @@ public Mono<FieldsDto> buildForm(
                                      @NonNull String defaultApiKey
     ) {
         return Mono.using(
-                () -> tracer.spanBuilder("NpgClient#buildForm")
-                        .setParent(Context.current().with(Span.current()))
-                        .setAttribute(NPG_CORRELATION_ID_ATTRIBUTE_NAME, correlationId.toString())
-                        .startSpan(),
-                span -> paymentServicesApi.pspApiV1OrdersBuildPost(
+                () -> {
+                    paymentServicesApi.getApiClient().setApiKey(defaultApiKey);
```

Comment:
why is there compared to the old openapi?

Reply:
Because current spec declares the API key as a `securityScheme` so it's not available anymore as a method parameter in the generated code